### PR TITLE
Add Initial Homepage

### PR DESCRIPTION
**Pull request by snkzt**
First commit to the main repositry

### CODE REVIEW

File: assets/css/home.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+#sign:hover {
+  color: black;
+  transition-duration: 0.25s;
+  text-shadow: none;
+}
+
+
+/* erasethislater {
```
Comment (VeysonD) [inline assets/css/home.css:43]: I'm curious on why this is commented out 🤔

File: assets/html/home.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="../css/home.css">
+    <!-- <meta name="viewport" content="width=device-width,initial-scale=1"> -->
```
Comment (VeysonD) [inline assets/html/home.html:5]: Same as above, I'm curious on why this is commented out 🤔

File: assets/css/home.css (hunk 1 above)
Comment (snkzt) [reply assets/css/home.css:43]: this is for reference

File: assets/css/sign.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  text-decoration: none;
+  color: white;
+  font-size: 20px;
+  font-style: italic;
+  margin: 3% 0 0 0;
+}
+
+#link:hover {
+  font-weight: bold;
+}
```
Comment (VeysonD) [inline assets/css/sign.css]: Let's add a new line after this

File: assets/css/home.css (hunk 1 above)
Comment (snkzt) [reply assets/css/home.css:43]: I do not need them anymore but might need them later

File: assets/html/sign.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="row">
+          <div class="col">
+            <p id="app-title">
+              WRITE YOUR WISH
+              <br>
+              　TO MAKE IT HAPPEN
+            </p>
+          </div>
+          <div class="col">
+            <div id=credit>
```
Comment (VeysonD) [inline assets/html/sign.html]: This has an id of credit, but it's not in quotation marks.  I'm not sure if this is getting the correct CSS

File: README.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ Manage Your Account
   - Click your name and update information you registered with 'Update My Info' on the drop down list 
   - Choose sign out from the drop down list to say bye for now
 
-### Built With
+### Build With
```
Comment (VeysonD) [inline README.md]: The previous one was correct:  `Built With`

File: assets/css/home.css (hunk 1 above)
Comment (VeysonD) [reply assets/css/home.css:43]: Ok, that's fine 👍

File: assets/html/home.html (hunk 2 above)
Comment (snkzt) [reply assets/html/home.html:5]: For this one, I want add slight changes after the main style has fixed

**Comment by VeysonD** [comment]
@snkzt Can you include a screenshot into your PR Description?

File: assets/html/home.html (hunk 2 above)
Comment (snkzt) [reply assets/html/home.html:5]: so now it is commented out and then applied later

File: assets/css/sign.css (hunk 3 above)
Comment (snkzt) [reply assets/css/sign.css]: ok

**Comment by VeysonD** [comment]
Let's also give this PR a title that is easy to read and simple enough to see at a glance

Maybe like:  "Add Initial Homepage" perhaps?  What do you think?

File: assets/html/sign.html (hunk 4 above)
Comment (snkzt) [reply assets/html/sign.html]: thanks! will fix!

File: assets/css/sign.css (hunk 3 above)
Comment (snkzt) [reply assets/css/sign.css]: ok done! new file will be pushed later

File: assets/html/sign.html (hunk 4 above)
Comment (snkzt) [reply assets/html/sign.html]: ok done! new file will be pushed later

File: README.md (hunk 5 above)
Comment (snkzt) [reply README.md]: ok done! new file will be pushed later

File: assets/html/home.html (hunk 2 above)
Comment (VeysonD) [reply assets/html/home.html:5]: ok sounds good to me

Usually I will leave a "TODO: blahblahblah" comment to remind myself to come back and finish what I started

**Review by snkzt — COMMENTED** [review]
image

**Comment by VeysonD** [comment]
The picture is fine, but in the future you can add screenshots by editing your initial comment in this PR

Anyway, this screenshot looks good to me!

**Comment by snkzt** [comment]
the first pic was home page

**Comment by snkzt** [comment]
This one is in progress, sign in/up page

**Comment by snkzt** [comment]
> The picture is fine, but in the future you can add screenshots by editing your initial comment in this PR
> 
> Anyway, this screenshot looks good to me!
Ah ok!

**Comment by VeysonD** [comment]
LGTM!

Merge this anytime you like

**Pull request merged**